oscovida/metadata.py

```diff
@@ -5,7 +5,7 @@
 import pandas as pd
 
 
-
+# FIXME: where should this come from?
 MetadataStorageLocation = os.path.join("coronavirus-metadata", "regions")
```

Comment:
The first part (`coronavirus-metadata`) is the name of the sudirectory in which we store (meta)data about each processed region as they are processed. So I would maybe rename it to (`oscovida-metadata`). I am not sure, `metadata` is the best name, but it may not be so important.

The files written in there are human-readable json (I think), so it is useful to label the directory meaningfully and not to hide it, I think (as we could do through a hidden subdirectory).